templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -143,7 +143,7 @@ below, and add more to your repertoire as needed.
 
 * **View image**: <kbd>V</kbd>
 
-* **Edit message**: <kbd>E</kbd>
+* **Edit message or view message source**: <kbd>E</kbd>
```

Comment:
@alya we might need more explanation or link here; I'm not sure users reading this will understand what "view message source" means here.